App/src/Controller/Group/GroupBalanceController.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+namespace App\Controller\Group;
+
+use App\Entity\Group;
+use App\Service\GroupExpenseBalancer;
+use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\Routing\Attribute\Route;
+
+class GroupBalanceController extends AbstractController
+{
+    #[Route('/groups/{id}/balance', name: 'groups_balance', methods: ['GET'])]
```

Comment:
Pour les méthodes, tu peux utiliser les constantes + ne pas mettre de tableau si jamais tu n'en a qu'une seule :
```suggestion
    #[Route('/groups/{id}/balance', name: 'groups_balance', methods: Request::METHOD_GET)]
```

Reply:
done